pages/styles/style.css

```diff
@@ -0,0 +1,186 @@
+:root {
+	--darkColor: #E04D01;
+	--lightColor: #f37225;
+	--primaryColor: #FEE2C5;
+	--secondaryColor: #2A2550;
+}
+
+.logo img {
+	height: 200px;
+	width: 200px;
+}
+
+.logo {
+	display: inline-flex;
+}
+
+.listnav a:hover{
+	background-color: yellow;
+  }
+
+.blogpostfoto {
+	width: 450px;
+	height: 450px;
+	padding-left: 10px;
+}
+
+
+figure img {
+	width: 150px;
+	height: 112px;
+}
+
+.avatar {
+	width: 250px;
+	height: 250px;
+}
+
+.containerbio {
+	display: inline-flex
+}
+
+
+.container {
+	/* Reminder You Set display to grid */
+	display: grid;
+	/* You setup blog columns */
+
+	grid-template-areas:
+		"hd hd hd hd hd hd hd hd"
```

Comment:
Since each row has the same grid area all the way across, I don't think it needs to be repeated.